google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -178,11 +178,8 @@ def project_id
         #   Flattens all nested and repeated fields in the query results. The
         #   default value is `true`. `large_results` parameter must be `true` if
         #   this is set to `false`.
-        # @param [Integer] maximum_billing_tier Limits the billing tier for this
-        #   job. Queries that have resource usage beyond this tier will fail
-        #   (without incurring a charge). Optional. If unspecified, this will be
-        #   set to your project default. For more information, see [High-Compute
-        #   queries](https://cloud.google.com/bigquery/pricing#high-compute).
+        # @param [Integer] maximum_billing_tier Deprecated: Change the billing
```

Comment:
This should be moved to the end of the list of arguments now that it is deprecated.